gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -97,9 +97,19 @@ export default function NxFormLayoutExample() {
 
   const [tagColorState, setTagColor] = nxFieldsetStateHelpers.useRadioGroupState<SelectableColor>(undefined, validator);
 
-  const [selectedTransferItems, setSelectedTransferItems] = useState<Set<number>>(new Set()),
+  const [transferItemsIsPristine, setTransferItemsIsPristine] = useState<boolean>(true),
+      [selectedTransferItems, setSelectedTransferItems] = useState<Set<number>>(new Set()),
       [availableTransferItemsFilter, setAvailableTransferItemsFilter] = useState(''),
-      [selectedTransferItemsFilter, setSelectedTransferItemsFilter] = useState('');
+      [selectedTransferItemsFilter, setSelectedTransferItemsFilter] = useState(''),
+      [transferItemsValidationErrors, setTransferItemsValidationErrors] = useState<string | null>(null);
+
+  function transferItemsOnChange(selectedItems: Set<number>) {
```

Comment:
Would it make sense to create another hook to manage this, similar to `useCheckboxGroupState`?

Reply:
> Would it make sense to create another hook to manage this, similar to `useCheckboxGroupState`?

Yup! I was about to ask about that in the meetings.

Reply:
> > Would it make sense to create another hook to manage this, similar to `useCheckboxGroupState`?
> 
> Yup! I was about to ask about that in the meetings.

Added the hook here: 8ccfdf27dbf9f71a7f1add5c57a6057dc62b2747

Reply:
> Added the hook here: https://github.com/sonatype/sonatype-react-shared-components/commit/8ccfdf27dbf9f71a7f1add5c57a6057dc62b2747

Actually I guess just like checkbox and radio fieldsets we should really start with pure-function state helpers (analogous to `checkboxGroupUserInput` etc) and then build the react hook on top of that.  We want to make things easily usable whether the validation is happening within a react component or somewhere else (like in redux)

Reply:
> > Added the hook here: [8ccfdf2](https://github.com/sonatype/sonatype-react-shared-components/commit/8ccfdf27dbf9f71a7f1add5c57a6057dc62b2747)
> 
> Actually I guess just like checkbox and radio fieldsets we should really start with pure-function state helpers (analogous to `checkboxGroupUserInput` etc) and then build the react hook on top of that. We want to make things easily usable whether the validation is happening within a react component or somewhere else (like in redux)

Added that here: 7caa43015d039e3e6d9f7740338ffaf2656ad6cc